controls/radfiledialogs/features/filtering.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Filtering
+page_title: Filtering
+description: This article will described the filter functionality of the RadFileDialogs.
+slug: radfiledialogs-features-filtering
+tags: filtering,features,radopenfiledialog,radsavefiledialog
+published: True
+position: 3
+---
+
+# Filtering
+
+With the R3 2017 SP 1 version of Telerik UI for WPF, you can now specify a filter to the __RadOpenFileDialog and RadSaveFileDialog__ dialogs in order indicate the user which types of files should be opening or saving. To enabled this functionality you can set the __Filter__ and __FilterIndex__ properties. These properties works the same way as the Microsoft file dialogs ([OpenFileDialog](https://msdn.microsoft.com/en-us/library/microsoft.win32.openfiledialog(v=vs.110).aspx), [SaveFileDialog](https://msdn.microsoft.com/en-us/library/microsoft.win32.savefiledialog(v=vs.110).aspx)) do.
+
+
+The __Filter__ is a property of type string that determines what type of files should be used in the combo box. On the other hand the __FilterIndex__ property can be used to gets or sets the index of the filter currently selected in a file dialog.
+> The index value of the first filter entry is 1. 
+
+The string set to the __Filter__ property is parsed in strict way. So when parsing is not successful, an ArgumentException is thrown. The exception string is the same as the Microsoft dialogs.
```

Comment:
I don't think it is necessary to tell that the message is the same as the MS dialogs. Instead leave only the error message.